### PR TITLE
Add line breaks to instructions  (#40425)

### LINKED ISSUE #40425: Break challenge instructions into paragraphs
**Describe your problem and how to reproduce it:**

Currently the instruction for  "Data Structures: Create a Circular Queue" challenge is cramped up in one paragraph.

> In this challenge we will implement a circular queue. The circular queue should provide enqueue and dequeue methods which allow you to read from and write to the queue. The class itself should also accept an integer argument which you can use to specify the size of the queue when created. We've written the starting version of this class for you in the code editor. When you enqueue items to the queue, the write pointer should advance forward and loop back to the beginning once it reaches the end of the queue. Likewise, the read pointer should advance forward as you dequeue items. The write pointer should not be allowed to move past the read pointer (our class won't let you overwrite data you haven't read yet) and the read pointer should not be able to advance past data you have written. In addition, the enqueue method should return the item you enqueued if it is successful; otherwise it will return null. Similarly, when you dequeue an item, that item should be returned and if you cannot dequeue an item you should return null.

This makes it difficult to understand and keep track of the challenge requirements. It would be better to break the instructions down into paragraphs. Something like:

>  In this challenge we will implement a circular queue. The circular queue should provide enqueue and dequeue methods which allow you to read from and write to the queue. The class itself should also accept an integer argument which you can use to specify the size of the queue when created. We've written the starting version of this class for you in the code editor

 > When you enqueue items to the queue, the write pointer should advance forward and loop back to the beginning once it reaches the end of the queue. The enqueue method should return the item you enqueued if it is successful; otherwise it will return null.

> Likewise, the read pointer should advance forward as you dequeue items. When you dequeue an item, that item should be returned and if you cannot dequeue an item you should return null.

> The write pointer should not be allowed to move past the read pointer (our class won't let you overwrite data you haven't read yet) and the read pointer should not be able to advance past data you have written

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/coding-interview-prep/data-structures/create-a-circular-queue

**Tell us about your browser and operating system:**
* Browser Name: N/A
* Browser Version: N/A
* Operating System: N/A

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by ghost**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40425

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md
Comment (nhcarrigan) [inline curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md:48]:
```suggestion
In this challenge we will implement a circular queue. The circular queue should provide `enqueue` and `dequeue` methods which allow you to read from and write to the queue. The class itself should also accept an integer argument which you can use to specify the size of the queue when created. We've written the starting version of this class for you in the code editor.
```
Nitpick here - missing period.

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md
Comment (nhcarrigan) [inline curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md:52]:
```suggestion
Likewise, the read pointer should advance forward as you dequeue items. When you dequeue an item, that item should be returned. If you cannot dequeue an item you should return `null`.
```
What do you think about splitting this into separate sentences?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Hey there,

Thanks for submitting this PR. I've got a couple of small tweaks to look at. 🙂

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md
Comment (RandellDawson) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md:52]: I would also add a comma between `item` and `you` in the second sentence.

Comment (nhcarrigan) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md:52]: Good catch - I updated the suggestion.

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md
Comment (ianturner88) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md]: Great! Good catch :)

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md
Comment (ianturner88) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-circular-queue.md]: Noted. I made the appropriate changes! Thanks :)

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires at least one additional approval before this can be merged. 🙂Additionally, we are currently in a code freeze and this cannot be merged until the freeze is lifted.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Congrats @ianturner88 on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**
